exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,9 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  return sentence.includes("code");
```

Comment:
Yes. Well done. sentence is of type string. String has a method called .includes() which checks if a substring (code) is contained in the string